extensions.yaml

```diff
@@ -43,10 +44,21 @@ extensions:
       - kernel-rt-modules
       - kernel-rt-modules-extra
       - kernel-rt-devel
-  # https://github.com/openshift/machine-config-operator/pull/2376/
+  # https://github.com/openshift/machine-config-operator/pull/2456
+  # https://github.com/openshift/enhancements/blob/master/enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md
   # GRPA-3123
-  qemu-kiwi:
+  # The dependency tree here is:
+  # - kata-containers (rhocp)
+  #   - qemu-kiwi (advanced-virt)
+  #     - ipxe-roms-qemu (appstream)
+  #     - libpmem (appstream)
+  #     - pixman (appstream)
+  #     - qemu-kvm-common (advanced-virt)
+  #     - seabios-bin (advanced-virt)
+  #     - seavgabios-bin (advanced-virt)
```

Comment:
Fine as is, I'm just not sure that anyone will really care about the bios level packages.  Calling out `kata-containers -> qemu-kiwi` though makes total sense to me.

Reply:
Fixed!